homotopy-web/src/model/proof.rs

```diff
@@ -167,7 +168,9 @@ impl Action {
     pub fn relevant(&self) -> bool {
         !matches!(
             self,
-            Action::HighlightSlice(_) | Action::HighlightAttachment(_)
+            Action::HighlightSlice(_)
+                | Action::HighlightAttachment(_)
+                | Action::EditSignature(SignatureEdit::Edit(_, _))
```

Comment:
This is not quite right. This means pressing `undo` will undo both the signature edit and whatever action came before that. I think it's better to leave it as it was.

Reply:
Yeah that makes sense, easy fix.